python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -273,7 +326,7 @@ def _get_concurrent_resource_demand_to_launch(
             # Enforce here max allowed pending nodes to be frac of total
             # running nodes.
             max_allowed_pending_nodes = max(
-                5, int(frac * running_nodes[node_type]))
+                5, int(self.upscaling_speed * running_nodes[node_type]))
```

Comment:
Please pull the `5` out into a `UPSCALING_INITIAL_NUM_NODES = 5` at the top of this file, with a comment.